### PR TITLE
Initial Login UI

### PR DESCRIPTION
**Pull request by kbrimm**
Implements UI for login page. 
Still needs: login implementation as well as links to account creation and password retrieval.

### CODE REVIEW

**Comment by SoySauceLabs** [comment]
Everything checks out, looks good!  **APPROVED!**

**Comment by SoySauceLabs** [comment]
...I can let Sapphire do the honors of hittin' merge, captain, my captain.

**Comment by logicplace** [comment]
My only qualm is using the British spelling for gray! Looks great tho, thx. Is there any significance to 640px out of curiosity?

**Comment by kbrimm** [comment]
No significance - it was just an arbitrary distinction between a small and large screen. And you can pry my spelling of grey from my cold, clammy hands. :wink:

File: js/login.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
```
Comment (logicplace) [inline js/login.js:1]: Inconsistent opening {, would prefer like "function() {" below.

Also, in jQuery using "$(function() {" is sufficient for creating a document ready event.

Comment (logicplace) [inline js/login.js:1]: Additionally, there's no copyright info in this file.

File: login.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<!--
```
Comment (logicplace) [inline login.html:1]: Comment should appear below the DOCTYPE, as per IE's craziness.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <title>Plantalytics Login</title>
+        
+        <!-- Includes -->
+        <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
+        <script src="http://code.jquery.com/jquery-2.0.0.js"></script>
+        <link rel="stylesheet" href="css/login.css"/>
+        <script type='text/javascript' src='js/login.js'> </script>
```
Comment (logicplace) [inline login.html:21]: Should use double quotes.

File: login.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                <br>
+            </p>
+            <p class="centerBlock">
+                <!-- Will connect with account singup page once implemented -->
+                <a href="#">Get Plantalytics</a>
+            </p>
+
+        </div>        
+    </body>
+</html>
```
Comment (logicplace) [inline login.html:57]: eof newline

File: js/login.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    $('#loginButton').click(function() {
+        /* This function to handle user login
+         * once backend support allows
+         */
+        window.location.href = "dashboard.html";
+        return false;
+    });
+});
```
Comment (logicplace) [inline js/login.js:9]: eof newline

File: css/login.css [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    padding-left: 1em;
+    padding-right: 1em;
+    color: beige;
+    font-size: 100%;
+}
+
+input.button:active {
+    border: darkolivegreen solid 2px;
+    background: dimgrey;
+}
```
Comment (logicplace) [inline css/login.css:120]: eof newline

**Pull request merged**